pyproject.toml

```diff
@@ -10,11 +10,10 @@ authors = [{name = "metatensor-models developers"}]
 
 dependencies = [
     "ase",
-    "torch",
```

Comment:
since the code is using torch as a dependency, I would keep it explicitly here. pip will then resolve it to a supported version when installing metatensor-torch.

Reply:
Currentlty at least it doesn't. I tried using "torch<2.3" and still I got an error that I have torch 2.3 installed. At least local on my machine.

Reply:
yes, the metatensor-torch wheels don't contain the right torch versions yet. But this should be fine here, since metatensor-torch v0.3.0 did not have wheel and we always built from sources